tdrs-backend/Pipfile

```diff
@@ -59,6 +59,7 @@ cerberus = "==1.3.4"
 xlsxwriter = "==3.1.9"
 openpyxl = "==3.1.2"
 sendgrid = "==6.10.0"
+django_prometheus = "==2.3.1"
```

Comment:
any concerns with this third party lib? i took a quick look through the repo and everything seems pretty good, it just doesn't appear to be officially maintained.

Reply:
Good question, everywhere I looked it was the dependency that everyone recommended/talked about. As far as I know there isn't a django plugin maintained by the prometheus community.

Reply:
Why not using python library instead? you will only have to add the line in middleware but it is well maintained